app/services/ingest_iati_activities.rb

```diff
@@ -67,6 +68,11 @@ def call
     new_activity.description = normalize_string(description)
   end
 
+  private def add_programme_status(legacy_activity:, new_activity:)
+    # This will need to be replaced once we have a bi-directional mapping between programme_status and IATI status
+    new_activity.programme_status = "Replace me"
```

Comment:
What happens if you try to edit an activity whose `programme_status` is set like this?

Reply:
That's a very good question, and honestly I don't have an answer. What do you reckon the best approach is?

Reply:
I think any activities where we know the IATI status, but not `programme_status` need to be flagged in a way that we know to go back and fix them later. So I think a data migration that sets this attribute to "Replace me" in cases where it is empty makes sense.

We probably should set the `programme_status` to the proper value in the cases where there is *only* one IATI status that matches it.

Any thoughts, @jcoglan ?

Reply:
I will create a new card for this so we can agree the best approach for this issue.